src/Piston/AtomActionResult.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Piston
```

Comment:
So looking at our favourite NuGetGallery they have [one custom action result in the project root](https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/ImageResult.cs) and [another in the Infrastrcture namespace](https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Infrastructure/SafeRedirectResult.cs). 

So do we go for Infrastucture? Leave where is? Something else? #bored